run.py

```diff
@@ -71,55 +71,85 @@ def install_secret_keys(application, secret_session='secret_key',
     if do_exit:
         sys.exit(1)
 
+        
 install_secret_keys(app)
 
 
 # Expose submenu method for jinja templates
 def sub_menu_open(menu_entries, active_route):
+    '''
```

Comment:
Please use double quotes instead of single quotes (conforming to PEP257). Also, I'm missing the param & type's for the arguments of the method.